<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="Navnpro" failures="1" tests="2" name="Test(failed)" time="0.9" errors="0" timestamp="2024-02-22T11:26:47 IST">
  <testcase classname="testng.listeners.SampleTest" name="failSample" time="0.02">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testng.listeners.SampleTest.failSample(SampleTest.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- failSample -->
  <testcase classname="testng.listeners.SampleTest" name="skipSample" time="0.008">
    <skipped/>
  </testcase> <!-- skipSample -->
  <testcase classname="testng.listeners.SampleTest" name="searchJava" time="0.0">
    <ignored/>
  </testcase> <!-- searchJava -->
  <testcase classname="testng.listeners.SampleTest" name="sample" time="0.0">
    <ignored/>
  </testcase> <!-- sample -->
</testsuite> <!-- Test(failed) -->
